src/components/include/media_manager/media_manager.h

```diff
@@ -36,16 +36,62 @@
 #include <string>
 #include "media_manager/media_manager_settings.h"
 #include "protocol/service_type.h"
+#include "utils/macro.h"  // for "DEPRECATED"
 namespace media_manager {
 
+/**
+ * @brief Describes different sampling options for PerformAudioPassThru.
+ */
+typedef enum {
+  /// Invalid sampling rate.
+  SR_INVALID = -1,
+  /// Sampling rate of 8000 Hz.
+  SR_8KHZ = 0,
+  /// Sampling rate of 16000 Hz.
+  SR_16KHZ = 1,
+  /// Sampling rate of 22050 Hz.
+  SR_22KHZ = 2,
+  /// Sampling rate of 44100 Hz.
+  SR_44KHZ = 3
+} SamplingRate;
+
+/**
+ * @brief Describes different quality options for PerformAudioPassThru.
+ */
+typedef enum {
```

Comment:
What is the reason of introducing new enums But not using MOBILE_API in media manager?

Reply:
Thanks, please refer to my comment https://github.com/smartdevicelink/sdl_core/pull/2184#issuecomment-644639701.

Reply:
I have updated MediaManager to use Mobile API enums. Please refer to my commit 8211590. Thanks!